src/examples/src/widgets/list/ListItemRenderer.tsx

```diff
@@ -0,0 +1,105 @@
+import { create, tsx } from '@dojo/framework/core/vdom';
+import List, { ListItem } from '@dojo/widgets/list';
+import Icon from '@dojo/widgets/icon';
+import states from './states';
+import icache from '@dojo/framework/core/middleware/icache';
+import Example from '../../Example';
+import {
+	createResourceTemplate,
+	createResourceMiddleware
+} from '@dojo/framework/core/middleware/resources';
+import Button from '@dojo/widgets/button';
+import * as exampleCss from './ListItemRenderer.m.css';
+
+const resource = createResourceMiddleware();
+const factory = create({ icache, resource });
+const template = createResourceTemplate<typeof states[0]>('value');
+
+export default factory(function ListItemRenderer({ id, middleware: { icache, resource } }) {
+	const listItemClasses = { '@dojo/widgets/list-item': { height: [exampleCss.rowHeight] } };
```

Comment:
can you remove these style overrides please 😬, if it doesn't look right when removed then i'd say the current default behaviour isn't good enough.